packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -722,11 +722,17 @@ class TimeInput extends React.Component {
       { 'initial-focus': this.state.secondInitialFocused },
     ]);
 
+    const formatDescriptionId = `terra-time-input-description-format-${uuidv4()}`;
```

Comment:
`formatDescriptionId` changes on every render with the way it is. Perhaps wrap uuidv2 with `useRef`.

```suggestion
    const formatDescriptionId = `terra-time-input-description-format-${useRef(uuidv4())}`;
```